manifests/base/redis/statefulset.yaml

```
@@ -0,0 +1,38 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: redis
+spec:
+  serviceName: redis
+  replicas: 1
+  template:
+    spec:
+      securityContext:
+        fsGroup: 1001
+      containers:
+        - name: redis
+          image: "redis/redis-stack:6.2.6-v9"
```

Same image, the difference is that there's no "developer tools" that are not needed for a server.
```suggestion
          image: "redis/redis-stack-server:6.2.6-v9"
```